osquery/config/config.cpp

```diff
@@ -208,6 +208,16 @@ class Schedule : private boost::noncopyable {
    */
   std::map<std::string, uint64_t> denylist_;
 
+  /**
+   * @brief Set of denylisted queries which have been debug logged.
+   *
+   * A set of denied queries which have been debug logged. When a query gets
+   * denied by watchdog, we would like for it to be logged before it expires.
+   * Since the schedule is hit frequently, we track which queries have been
+   * logged as not to spam the debug logs.
```

Comment:
```suggestion
   * A set of denied queries which have been logged. When a query gets
   * denied by watchdog, we would like for it to be logged before it expires.
   * Since the schedule is hit frequently, we track which queries have been
   * logged as not to spam the logs.
```